File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = []  
    words = s.split(" ")
    
    for word in words:
        new_word = "" 
        for i, v in enumerate(word):
            if i % 2:
                new_word += v.lower()
            else:
                new_word += v.upper()
                
        answer.append(new_word)
    
    return " ".join(answer)

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    var answer = '';
    let cnt = 0;
    
    for (let i of s) {
        if (i == " ") {
            answer += " ";
            cnt = 0;
            continue;
        }
        
        if (cnt % 2) {
            answer += i.toLowerCase();
            cnt ++;
        } else {
            answer += i.toUpperCase();
            cnt ++;
        }

    }
    
    return answer;
}